lmdk/pic/smart_amp_test/smart_amp_test.toml

```diff
@@ -0,0 +1,80 @@
+version = [3, 0]
```

Comment:
We should really run the pre-processor on the toml files like we do with linker scripts. @kv2019i fyi.

Reply:
@lgirdwood @aiChaoSONG Hmm, given we always have one platform and one algo bit, not sure we need to go that far. But for sure this is one option for https://github.com/thesofproject/rimage/pull/176 One benefit is that we'll have cpp on all platforms, so this would cover the portability part.

Reply:
so upgrading to use cpp here would also be easy enough to do and also allow us to pick UUIDs and memory addreses and sizes from headers (instead of coding them more than once).